hacs.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "multimatic",
+  "domains": ["vrc", "vaillant", "multimatic"]
+}
```

Comment:
`zip_release` should be set to true, but then, I'm not sure on what to set in `filename` ? Zip in release are named: `vaillant-component-{release}`, so this is not static. Do you have any idea ?

`iot_class` should be `Cloud Polling`

Reply:
Other projects use `zip_release` when there's a built zip attached to the GitHub release, e. g. [alexa media player](https://github.com/custom-components/alexa_media_player/releases). In this repo, there's no such zip file attached. So, I don't think `zip_release` is applicable in this case.

Reply:
ha yeah, misunderstood the doc, sorry